### PR TITLE
Add build actions

### PR DESCRIPTION
**Pull request by andersjohnsen**

### CODE REVIEW

**Comment by jstroem** [comment]
Rechecking CLA status: @cla-bot check.

**Review by jstroem — APPROVED** [review]
LGTM

File: .github/workflows/build_macos.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Build macos
```
Comment (kasperl) [inline .github/workflows/build_macos.yml:1]: macOS

**Review by kasperl — APPROVED** [review]
Looks great!

**Review by floitsch — APPROVED** [review]
LGTM with changes.

**Pull request merged**